common-tasks/working-with-documents/en.md

```diff
@@ -4,8 +4,15 @@ title = "Working with Documents"
 
 # Working with Documents
 
-Solus comes pre-installed with LibreOffice, an office suite that is capable of replacing Microsoft Office for the most common tasks.
-Unless you are a heavy power user, LibreOffice should provide you with all the tools needed to solve your daily office challenges.
-LibreOffice Writer users may want to install the package `libreoffice-common-dictionaries` from Software Center to automatically enable spell-checking support.
+Solus comes pre-installed with LibreOffice, an office suite that is capable of replacing Microsoft<sup>TM</sup> Office for the most common tasks. By default, Solus provides Writer (Word replacement), Calc (Excel replacement), and Impress (PowerPoint replacement). Other applications such as Math can be installed from the Software Center. 
```

Comment:
> By default, Solus provides Writer (Word replacement), Calc (Excel replacement), and Impress (PowerPoint replacement). Other applications such as Math can be installed from the Software Center.

Should be on a new line otherwise we're going to have a long summary in the Common Tasks section.

Also, all the bits like Word, Excel, PowerPoint now need to be added with the appropriate trademark information, so I'd rather we refer to Writer as a "document editor", Calc as a "spreadsheet editor", Impress as a "presentation editor".